src/quo/components/token.cljs

```diff
@@ -0,0 +1,53 @@
+(ns quo.components.token
```

Comment:
maybe we can have this component in a specific namespace and put icon component there too? (we could move icon in a separate pr) 
quo.components.utitlites or something? perhaps there's a better name? the idea being to have a clearer separation that these files are not a direct DS component - also ha